js/apps/admin-ui/src/user-federation/ldap/LdapSettingsSearching.tsx

```diff
@@ -403,6 +404,38 @@ export const LdapSettingsSearching = ({
             )}
           ></Controller>
         </FormGroup>
+        <FormGroup
+          label={t("referral")}
+          labelIcon={
+            <HelpItem helpText={t("referralHelp")} fieldLabelId="referral" />
+          }
+          fieldId="kc-referral"
+        >
+          <Controller
+            name="config.referral[0]"
```

Comment:
I do believe newer versions of React Hook Form prefer the following syntax:
```suggestion
            name="config.referral.0"
```

Reply:
Done!